content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -121,7 +121,7 @@ token,user,uid,"group1,group2,group3"
 
 When using bearer token authentication from an http client, the API
 server expects an `Authorization` header with a value of `Bearer
-THETOKEN`.  The bearer token must be a character sequence that can be
+token`.  The bearer token must be a character sequence that can be
```

Comment:
suggestion to show to replace the value with the actual token
```suggestion
<token>`.  The bearer token must be a character sequence that can be
```

Reply:
Good idea, thanks :)